custom-plugins/fictive_studios/admin/fashion-designs/listing/fashion-designs-listing-helper.php

```diff
@@ -22,11 +23,8 @@ public function get_columns()
         return array(
             'cb' => '<input type="checkbox"/>',
             'name' => __('Name', 'name'),
-            'design_layer_1' => __('Layer 1', 'design_layer_1'),
-            'design_layer_2' => __('Layer 2', 'design_layer_2'),
-            'design_layer_3' => __('Layer 3', 'design_layer_3'),
-            'design_layer_4' => __('Layer 4', 'design_layer_4'),
-            'design_layer_5' => __('Layer 5', 'design_layer_5'),
+            'no_of_layers' => __('No of Layer', 'design_layer_1'),
```

Comment:
I don't think we need to have column for the count. We could just take count in MySQL using a subquery.